images/operator-registry.yml

```diff
@@ -1,11 +1,14 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/operator-framework/operator-registry
 content:
   source:
+    dockerfile: operator-registry.Dockerfile
     git:
       branch:
         target: release-{MAJOR}.{MINOR}
-      url: git@github.com:operator-framework/operator-registry.git
-    pkg_managers:
-    - gomod
```

Comment:
just a note for future viewers... cachito didn't like something about the repo change so to get it building this is now using legacy module spec. cachito owners were made aware, may have a fix soon; test before switching back to cachito.